brave_testrun_generator.py

```diff
@@ -651,293 +515,327 @@ def android_testruns(milestonever):
-    print(bigline)
+    print("--------------------------------------------------------")
+
+    print("Linux Checklist:")
+    print(linux_template)
     print("")
-    linTitle = "Manual test run on Linux for " + milestonever
+    linTitle = "Manual test run on Linux for " + tor_rel
     linList = ["OS/Linux",
                "release-notes/exclude",
                "tests",
                "QA/Yes",
-               "OS/Desktop"]
+               "OS/Desktop",
+               "feature/tor"]
 
     if args.test is None:
-        laptop_repo.create_issue(title=linTitle,
-                                 body=bigline,
-                                 assignee="srirambv",
-                                 milestone=laptop_milestone[milestonever],
+        bc_repo.create_issue(title=linTitle,
+                                 body=linux_template,
+                                 milestone=bc_milestone[tor_rel],
                                  labels=linList)
 
     return 0
 
+print("\n#######################################################################"
+      "###################################################")
+print("\n For Desktop or Android minor CR bump use the HF selection to "
+      "generate testruns\n")
+
+length = 0
+print("Current open milestones for Desktop/Android")
+while length < len(laptop_key):
+    print(str(length+1) + ". " + sorted(bc_milestone.keys())[length])
+    length += 1
+
+ioslength=0
+print("\nCurrent open milestones for iOS")
+while ioslength < len(ios_key):
+    print(str(ioslength+1) + ". " + sorted(ios_milestone.keys())[ioslength])
+    ioslength += 1
+
+print("\n Note: For Tor Release make sure you type \"Tor\" or \"tor\" instead "
+      "of the number")
+
+print("#######################################################################"
+      "###################################################")
 
-print("*********************************************************************"
-      "*****************************************************")
-print("                         Few things to check before generating test"
-      "runs")
-print("")
-print("1. Ensure all closed items have QA/Yes and QA/No labels added to"
-      "include in the test run")
-print("")
-print("2. For Laptop - Test runs will be generated upto 5 milestones")
-print("")
-print("3. Manually change ""milestone.keys"" value to generate for a specific"
-      "milestone on any platform")
-print("***********************************************************************"
-      "***************************************************")
-
-header = print("Create test runs for:\n")
+header = print("\nCreate test runs for:\n")
 laptop = print("1. Desktop Release")
-laptop_per = print("2. Desktop Per-release Checklist")
-laptop_hf = print("3. Desktop CR minor bump")
-ios = print("4. iOS")
-android = print("5. Android")
-cryptowallet = print("6. Brave Crypto Wallet - Ethereum Client\n")
+laptop_hf = print("2. Desktop Hotfix")
```

Comment:
Suggest renaming `Desktop Hotfix` --> `Desktop Reduced Manual Pass` (or similar).

Rationale - there are Desktop releases (ex. 1.26.x) which are not hot fixes and we do the reduced manual passes.